packages/botonic-api/src/data-provider/dynamodb-utils.ts

```diff
@@ -13,6 +13,10 @@ export const SORT_KEY_NAME = 'SK'
 export const USER_PREFIX = 'USR#'
 export const EVENT_PREFIX = 'EVT#'
 
+function capitalize(string: string) {
+  return string.charAt(0).toUpperCase() + string.slice(1)
```

Comment:
As a general string method, you should protect the empty string case

Reply:
good catch! fixed! 👍